sample-new-architecture/android/app/build.gradle

```diff
@@ -62,7 +62,7 @@ project.ext.sentryCli = [
     collectModulesScript: "../../../dist/js/tools/collectModules.js",
     modulesPaths: [
       "node_modules",
-      "../..",
+      "../node_modules",
```

Comment:
I agree `../node_modules` makes sense to add, but `../..` should stay to pick up the `@sentry/react-native` package.